extensions/twenty/src/helper/formatValues.ts

```diff
@@ -30,6 +30,11 @@ export function formatValues(values: Record<string, any>, objectRecordMetadata:
           }
           break;
         }
+        case "RATING": {
+          if (formattedValues[key] === "") {
+            formattedValues[key] = null;
+          }
```

Comment:
- add break; statement, this also gives error in CI, run `npm run lint` and fix all lint errors as suggested.
```suggestion
          }
          break;
```